tools/common_lib/src/layers_utils.h

```diff
@@ -351,11 +355,75 @@ inline void fill_with_constant_linear_container_half(std::span<std::byte> contai
     }
 }
 
+inline uint8_t clamp_value(uint8_t value, uint8_t min, uint8_t max)
+{
+    if (value < min) return min;
```

Comment:
Wouldn't be std::clamp(...) enough? Doc: https://en.cppreference.com/w/cpp/algorithm/clamp

Reply:
Changed